Postman Test Suite/Ed-Fi ODS-API Integration Test Suite GetByExample Tests.postman_collection.json

```diff
@@ -1950,6 +1952,206 @@
 							"response": []
 						}
 					]
+				},
+				{
+					"name": "When filtering by IdentificationCode",
```

Comment:
I feel like these tests could really use some negative test data (data that would show up in the results if the implementation was incorrect).  For example, in a "Setup" folder create a new temporary School that has identification codes that match the value being searched but uses a different identification system. (Also add a "Teardown" folder that removes this School and use a separate subfolder in between to hold the test requests, as necessary).

So, for example, create a School with an identification code that uses `uri://ed-fi.org/EducationOrganizationIdentificationSystemDescriptor#Other` with a value of `255901001`. This school should also be returned when only searching by identification code `255901001` but should not be returned if both the identification system and code are used in the search.

![image](https://github.com/user-attachments/assets/3c515dc5-586b-422e-b3af-81169220bfd3)